cltk/tokenize/sentence.py

```diff
@@ -75,6 +85,9 @@ def _setup_tokenizer(self, tokenizer: object):
         params = tokenizer.get_params()
         return PunktSentenceTokenizer(params)
 
+    def _get_models_path(self: object, language): # pragma: no cover
```

Comment:
you may want a test for this, since if the dir isn't found an could be thrown, etc.

Reply:
Moved the call into the 'try' block so that it would raise a error about missing models. Was reading this morning about whether or not 'private' methods should be unittested—seemed like the consensus was no, but happy to hear otherwise if you think they should. (Not just here—you explained the reason why clearly enough in this case.)

Reply:
I would unittest them, and it's an easy place to establish free coverage points

Reply:
I totally agree with testing private methods. The label private is semantics more than anything -- despite the name, they can still be used/abused by an API consumer.